runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -6376,9 +6374,9 @@ makeMemCpyByteToCharGraph(TR::Compilation *c, int32_t ctrl)
    }
 
 
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
+
+
+
```

Comment:
Lines that contain only slashes will be ignored by Doxygen, so there's no need to change these lines. May I ask you to change them back to the following?

```
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
```